cmd/wallet-server/startcmd/start.go

```diff
@@ -129,6 +130,11 @@ const (
 	hubAuthURLFlagName  = "hub-auth-url"
 	hubAuthURLFlagUsage = "Hub Auth Servr URL"
 	hubAuthURLEnvKey    = "HTTP_SERVER_HUB_AUTH_URL"
+
+	validateDevicesFlagName  = "validate-devices"
+	validateDevicesFlagUsage = "Enable webAuthn device credential validation." +
+		" Alternatively, this can be set with the following environment variable: " + validateDevicesEnvKey
+	validateDevicesEnvKey = "VALIDATE_DEVICE_CREDENTIALS"
```

Comment:
I believe this shouldn't be optional.

Reply:
done

Reply:
@Moopli sorry I wasn't clear.

What I mean is - I believe there is no need for this command line parameter because the validation should always be done.

Reply:
I've updated the PR, the validation is now always done, and there's no option to disable it.